pkg/resource/plugin/plugin.go

```diff
@@ -211,7 +211,7 @@ func newPlugin(ctx *Context, pwd, bin, prefix string, args, env []string) (*plug
 							// The server is unavailable.  This is the Linux bug.  Wait a little and retry.
 							time.Sleep(time.Millisecond * 10)
 							continue // keep retrying
-						case codes.Unimplemented, codes.ResourceExhausted:
+						case codes.Unimplemented, codes.ResourceExhausted, codes.Internal:
```

Comment:
grpc-js returns this code for a non-existent rpc entrypoint

Reply:
Won't this be a breaking change with respect to down-level CLIs? Those CLIs would not be able to use the new SDK.

Reply:
Yes, this would be.